src/Fixer/Alias/SetTypeToCastFixer.php

```diff
@@ -155,7 +155,7 @@ protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
     }
 
     /**
-     * @return list<list<int>>
+     * @return list<array{int, int, int}>
```

Comment:
OK, but i would rather convert tuple to dto {functionName, $openParenthesis, $closeParenthesis}

Reply:
But this should be done in `AbstractFunctionReferenceFixer` and  requires changes to all subclasses.
Is it ok to only fix the phpdoc here for now?